src/browser/transports/socket_server.js

```diff
@@ -4,16 +4,18 @@ Copyright 2017 OpenFin Inc.
 Licensed under OpenFin Commercial License you may not use this file except in compliance with your Commercial License.
 Please contact OpenFin Inc. at sales@openfin.co to obtain a Commercial License.
 */
-let http = require('http');
-let EventEmitter = require('events').EventEmitter;
-let util = require('util');
+const http = require('http');
+const EventEmitter = require('events').EventEmitter;
+const util = require('util');
+const log = require('../log');
```

Comment:
Wouldn't it be nice, sooo nice, to have `import`s here, since these lines are being modified and there is this one beautiful `import` following these changes? :)